common/common_test.go

```diff
@@ -67,6 +67,23 @@ func TestLoadBuildConfigFromProvenance(t *testing.T) {
 	checkBuildConfig(config, t)
 }
 
+func TestParseBuilderImageURI(t *testing.T) {
```

Comment:
Nice! :) 

I was thinking if a test failing to parse an imageURI makes sense, too, but I am not too convinced myself.

Reply:
Good point. Added a negative test.